src/modules/chat_state/index.js

```diff
@@ -30,14 +28,9 @@ function displaySeconds(s) {
 
 function loadHTML() {
     const $stateContainer = $(`#${CHAT_STATE_ID}`);
-    const $headerSelector = $(CHAT_HEADER_SELECTOR);
-    const $chatSendButtonSelector = $(CHAT_SEND_BUTTON_SELECTOR);
-    if (!$headerSelector.length || $headerSelector.find(CHAT_HEADER_PRIVATE_ROOM_SELECTOR).length) {
```

Comment:
the header selector is correct code to handle removing the state indicators when navigating to a private room